### ISSUE TITLE
Page becomes unresponsive while rendering backlinks

### ISSUE THREAD

**Issue by unqueued** [lostintangent/wikilens#34]
On the latest versions of VSCode on Windows and macOS, this consistently happens.

I tried it with no other extensions.

I'm not sure exactly when this problem was introduced, or if anyone else is experiencing it.

But if a page would have backlinks, it appears to get stuck in a loop with an empty comment and then backlinks.

![output](https://github.com/lostintangent/wikilens/assets/5961515/d9f510dc-d57d-4ce3-92ae-c7a0748bda05)

**Comment by NdagiStanley** [comment]
I'm experiencing the same issue.
I've just installed the extension.

Any fix?

**Comment by unqueued** [comment]
@NdagiStanley  Yes, see my linked PR which fixes it: https://github.com/lostintangent/wikilens/pull/35

**Comment by unqueued** [comment]
@NdagiStanley  until it gets merged, you can build from my fork: https://github.com/unqueued/wikilens

In order to build with newer version of node, you need to do:
```

git clone https://github.com/unqueued/wikilens
cd wikilens
npm install
NODE_OPTIONS='--openssl-legacy-provider' npm package

```

The you can manually install the `wikilens-0.0.6.vsix` file into VSCode.

**Comment by NdagiStanley** [comment]
Great. Thank you, @unqueued. Also, thank you for the instructions, too.
It works as expected now.

cc @lostintangent

**Comment by NdagiStanley** [comment]
> ![](https://user-images.githubusercontent.com/5961515/260880634-d9f510dc-d57d-4ce3-92ae-c7a0748bda05.gif)

@unqueued What did you use to make this gif?

**Comment by unqueued** [comment]
@NdagiStanley I used arcsine.chronicler https://github.com/arciisine/vscode-chronicler

**Comment by NdagiStanley** [comment]
Noted
Thank you.
